src/cmdlib.sh

```diff
@@ -425,21 +425,23 @@ for line in sys.stdin:
     lockfile["packages"][name] = {"evra": f"{evr}.{arch}"}
 json.dump(lockfile, sys.stdout)' > "${local_overrides_lockfile}"
 
-        < "${tmp_overridesdir}/pkgs.txt" python3 -c '
+        < "${tmp_overridesdir}/pkgs.txt" python3 -c "
```

Comment:
It feels a little awkward (makes it hard to read) to lead the command with the redirection. Maybe we either use a useless use of cat (`cat "${tmp_overridesdir}/pkgs.txt" | python3 -c`) or we add the redirection at the bottom:

```
yaml.dump(manifest, sys.stdout)" < "${tmp_overridesdir}/pkgs.txt" >> "${override_manifest}"
```